### PR TITLE
Tidied up page titles

### LINKED ISSUE #50: Title should display correctly on listing page
Currently it appears as "Loading...".

### PR DESCRIPTION
**Pull request by voodoochild**
Capitalization and formatting of content title ([name] | [type]) can be dropped if preferred.

Fixes #50.

### CODE REVIEW

**Comment by tivac** [comment]
I'd prefer if this used `lodash.capitalize`, I know it's a bit pedantic but there's no reason for us to own that code.

Otherwise LGTM.

**Comment by voodoochild** [comment]
TIL `lodash.capitalize` is a thing!

**Comment by tivac** [comment]
https://devdocs.io/lodash/ is a wonderful, valuable resource.

File: src/pages/content-edit.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ var m      = require("mithril"),
     update   = require("../lib/update"),
     watch    = require("../lib/watch"),
 
+    capitalize = require("lodash.capitalize"),
```
Comment (tivac) [inline src/pages/content-edit.js]: Gonna be a butt and ask you to move this up w/ all the other `node_modules` declarations.

**Comment by voodoochild** [comment]
Oh, I _love_ lodash, but didn't realise that they had a util function for this. Guess it'll have to become the first place I check from now on.

File: src/pages/content-edit.js (hunk 1 above)
Comment (voodoochild) [inline src/pages/content-edit.js]: WTB butt emoji.

**Comment by tivac** [comment]
Yay thanks :smile_cat:

**Pull request merged**